src/drake/core.clj

```diff
@@ -106,8 +106,13 @@
    it. This is safe to do since it's the default filesystem,
    but it gives us a bit easier compatibility with existing tools."
   [filename]
-  (let [n (dfs/normalized-path filename)]
```

Comment:
You know, I thought about this some more. Why would you ever NOT want a function to correctly handle optional files? Why even have a ```normalize-filename-for-run*``` that fails for optional files?

Also in the case of functions like ```parser/add-prefix``` there there are other arguments besides a file name, this wrapping breaks down as we have no way of telling which argument is the file name.

I'm open to improving this in another way if you have any other ideas though.